src/Spice86.Core/Emulator/CPU/CfgCpu/Ast/Operations/BinaryOperation.cs

```diff
@@ -1,3 +1,23 @@
 ﻿namespace Spice86.Core.Emulator.CPU.CfgCpu.Ast.Operations;
 
-public enum BinaryOperation { PLUS, MULTIPLY, EQUAL, NOT_EQUAL, ASSIGN }
\ No newline at end of file
+public enum BinaryOperation { 
```

Comment:
Make sure those are implemented in AstExpressionBuilder and AstInstructionRenderer.\
Same remark for UnaryOperation.